src/main/java/core/turnorders/TurnOrder.java

```diff
@@ -166,14 +167,25 @@ public void endRound(AbstractGameState gameState) {
 
         roundCounter++;
         if (nMaxRounds != -1 && roundCounter == nMaxRounds) {
-            gameState.setGameStatus(GAME_END);
```

Comment:
Related to comment of that functionality that is better on the FM. The initial idea was that this would just set the game_end flag in the game state, then the forward model would process the end of game appropriately.

Reply:
"The initial idea was that this would just set the game_end flag in the game state, then the forward model would process the end of game appropriately."

I really don't like that idea. It seems to introduce the opportunity of bugs in the gap between setting the flag and actually implementing stuff. Are there any use cases where we can;t just end the game *immediately* it's over? To discuss.